main.py

```diff
@@ -23,6 +25,7 @@ def setup(self):
         """Initialize the NApp's links list."""
         self.links = {}
         self.store_items = {}
+        self.link_up_timer = getattr(settings, 'LINK_UP_TIMER', 10)
```

Comment:
I know this code was my own suggestion (thanks for accepting it! :-), but now I have the feeling that this hard-coded constant should be declared as a global (`DEFAULT_LINK_UP_TIMER`, for example).